dizoo/d4rl/config/hopper_medium_expert_ibc_ar_config.py

```diff
@@ -0,0 +1,62 @@
+from easydict import EasyDict
+
+cuda = True
+
+main_config = dict(
+    exp_name='hopper_medium_expert_ibc_ar_seed0',
+    env=dict(
+        env_id='hopper-medium-expert-v0',
+        evaluator_env_num=8,
+        use_act_scale=True,
+        n_evaluator_episode=8,
+        stop_value=6000,
+    ),
+    policy=dict(
+        cuda=cuda,
+        model=dict(
+            type='arebm',
+            import_names=['ding.model.template.ebm'],
```

Comment:
it is not necessary to add type and import_names in main_config

Reply:
Autoregressive ebm is not the default model in my implementation, so maybe we have to add `type` and `import_names` here if we do need to use `arebm`.

Reply:
I got you. File is changed